src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1127,6 +1127,15 @@ public String execute(Jedis connection) {
       }
     }.run(null);
   }
+  @Override
+  public String readonly() {
```

Comment:
It's only there due to JedisCluster implementing BasicCommands in master. The cluster-revised option is much cleaner. In the meantime, we could stub it out to return OK. Since JedisCluster will always send that to a master node, there is really no reason to call readonly. But if someone did, a OK stub would result in their next command working as expected.